en/mkdocs.yml

```diff
@@ -268,9 +268,20 @@ nav:
     - 'EMail OTP Configurations': guides/mfa/email-otp-config-advanced.md
     - 'SMS OTP Configurations': guides/mfa/sms-otp-config-advanced.md
     - 'TOTP Configurations': guides/mfa/totp-config-advanced.md
+  - 'Admin Portal':
+      - 'Applications':
+          - admin-portal/applications/overview.md
+          - 'Add New Application':
```

Comment:
Shall we add another section above to explain about listing ?

Reply:
Ayye overview page contains information on listing.

Reply:
Ok. Then it will be ok